src/config.rs

```diff
@@ -0,0 +1,61 @@
+use configster;
+
+#[derive(Debug, PartialEq)]
+pub struct Data {
+    pub realname: String,
+    pub nick: String,
+    pub password: String,
+    pub server_addr: String,
+    pub server_port: u32,
+}
+
+impl Data {
+    pub fn new() -> Self {
```

Comment:
Seems like unnecessary overhead to construct and then populate rather than doing it in a single operation.

```suggestion
    pub fn load(path: &str) -> Self {
```